The release artifact contains native changes, which cannot be applied with a patch

```
Run shorebird patch android
⠋ Fetching apps...
✓ Fetching apps (0.3s)
⠋ Building patch...
✓ Building patch (392.9s)
⠋ Detecting release version...
✓ Detected release version 2.2.0+1 (1.1s)
⠋ Fetching releases...
✓ Fetching releases (0.2s)
⠋ Fetching release artifacts...
✓ Fetching release artifacts (0.2s)
⠋ Fetching aab artifact...
✓ Fetching aab artifact (57ms)
⠋ Downloading release artifacts...
✓ Downloading release artifacts (1.1s)
⠋ Verifying patch can be applied to release...
✓ Verifying patch can be applied to release (1.6s)
[WARN] The release artifact contains native changes, which cannot be applied with a patch.
    Changed files:
        base/dex/classes.dex
Exiting.
Error: Process completed with exit code 70.
```

I don't have this warning when running the patch from local. What's the issue and how to fix it?

--- a/packages/shorebird_cli/lib/src/patch_diff_checker.dart
+++ b/packages/shorebird_cli/lib/src/patch_diff_checker.dart
@@ -97,6 +97,11 @@ class PatchDiffChecker {
           yellow.wrap(
             archiveDiffer.nativeFileSetDiff(contentDiffs).prettyString,
           ),
+        )
+        ..info(
+          yellow.wrap('''
+
+If you don't know why you're seeing this error, visit our troublshooting page at ${link(uri: Uri.parse('https://docs.shorebird.dev/troubleshooting#unexpected-native-changes'))}'''),
         );
 
       if (!force) {

--- a/packages/shorebird_cli/test/src/patch_diff_checker_test.dart
+++ b/packages/shorebird_cli/test/src/patch_diff_checker_test.dart
@@ -154,6 +154,14 @@ void main() {
           verify(
             () => logger.info(yellow.wrap(nativeDiffPrettyString)),
           ).called(1);
+          verify(
+            () => logger.info(
+              any(
+                that:
+                    contains("If you don't know why you're seeing this error"),
+              ),
+            ),
+          ).called(1);
         });
 
         test('prompts user if force is false', () async {

